node-unix-interface.md

```diff
@@ -0,0 +1,11 @@
+# The Node-Unix interface
+
+Imagine being able to build scripts that can fit into any pipeline of Unix
+commands at any place. Not only is that possible with Node.js, it's easy! Node
+provides easy access to everything you need to create powerful system level
+utilities, and following the Unix philosophy of modularity and reusability
```

Comment:
I think so... it's been a while. :)
Re-reading the abstract I think that is was I was suggesting.